spec/factories/podcasts.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :podcast do
+    created_at { Time.now.utc }
```

Comment:
```suggestion
```

I'm not sure you need `created_at` in your factories.

Do the tests pass with this line?